### DISCUSSION TITLE
Sveltekit +page.ts and useFilenamingConvention rule

### DISCUSSION THREAD

**Discussion by minht11** [biomejs/biome#2244] [Q&A]
Sveltekit routes are defined something with + prefix something like "routes/+page.ts", trying to use Biome useFilenamingConvention rule seems to be incompatible

```json
"useFilenamingConvention": {
	"level": "error",
	"options": {
		"filenameCases": ["kebab-case", "export", "PascalCase"]
	}
}
```

**Comment by Conaclos** [comment]
Have you some docs reference for this? We could add a new option that whitelist some prefixes like `+`.

**Reply by minht11** [reply]
Docs for Sveltekit routes naming can be found [here](https://kit.svelte.dev/docs/routing#page). It's only `+` as far as I am aware.

**Reply by Conaclos** [reply]
We could also just accept a `+` prefix in any circonstance. Are you aware of other frameworks that use this ocnvention?

**Reply by minht11** [reply]
To my knowledge Sveltekit is the only one.

**Reply by spdiswal** [reply]
[Vike](https://vike.dev/route) uses the same convention with the `+` prefix.

**Comment by Conaclos** [comment]
I opened the issue #2341.
